<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>processSelector</class>
 <widget class="QDialog" name="processSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Process Tree</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="processTree" name="treeWidget">
     <property name="autoExpandDelay">
      <number>1</number>
     </property>
     <column>
      <property name="text">
       <string>ProcessID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Path</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="activateBtn">
       <property name="text">
        <string>Activate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="activateCloseBtn">
       <property name="text">
        <string>Activate + Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>processTree</class>
   <extends>QTreeWidget</extends>
   <header>processtree.h</header>
   <slots>
    <slot>activateClicked()</slot>
    <slot>activateCloseClicked()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>activateCloseBtn</sender>
   <signal>clicked()</signal>
   <receiver>treeWidget</receiver>
   <slot>activateCloseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>activateBtn</sender>
   <signal>clicked()</signal>
   <receiver>treeWidget</receiver>
   <slot>activateClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
